alibi_detect/models/tensorflow/losses.py

```diff
@@ -1,7 +1,8 @@
 import tensorflow as tf
-from tensorflow.keras.layers import Flatten
-from tensorflow.keras.losses import kld, categorical_crossentropy
 import tensorflow_probability as tfp
+from tensorflow.keras.losses import kld, categorical_crossentropy
```

Comment:
Uber pendantic nitpick, could just remove these changes and changes to `test_losses_tf.py`...

Reply:
i'm not sure I understand?